### PR TITLE
Create SQL native Gems docs

### PR DESCRIPTION
**Pull request by alexanderahn**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208011943581293

### CODE REVIEW

**Comment by alexanderahn** [comment]
To add to Transformations table once merge SQL transform and join PR.

File: docs/SQL/gems/transformations/deduplicate.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+| :--------- | :-------- |
+| John       | Doe       |
+| Alice      | Johnson   |
+| Bob        | Smith     |
+
+For `Unique Only`, the interim data will look like the following:
+
+| First_Name | Last_Name |
+| :--------- | :-------- |
+| Bob        | Smith     |
```
Comment (prashantpiyush) [inline docs/SQL/gems/transformations/deduplicate.md:83]: for unique_only/first/last case, the interim data will contain all columns irrespective what columns were selected

File: docs/SQL/gems/transformations/img/deduplicate_row_to_keep.png
Comment (prashantpiyush) [inline docs/SQL/gems/transformations/img/deduplicate_row_to_keep.png:1]: FYI: the UI will be updated based on product feedback https://app.asana.com/0/1201259583869898/1208546147028910/f

File: docs/SQL/gems/transformations/deduplicate.md (hunk 1 above)
Comment (alexanderahn) [reply docs/SQL/gems/transformations/deduplicate.md:83]: So it would look the same as the table on line 50?

Comment (prashantpiyush) [reply docs/SQL/gems/transformations/deduplicate.md:83]: same as line 63

Comment (prashantpiyush) [reply docs/SQL/gems/transformations/deduplicate.md:83]:
```
| First_Name | Last_Name | Type  | Contact       |
| :--------- | :-------- | :---- | :------------ |
| Bob        | Smith     | email | bob@smith.com |
```

Comment (prashantpiyush) [reply docs/SQL/gems/transformations/deduplicate.md:83]: @alexanderahn yes, based on your edited comment 😅

File: docs/SQL/gems/transformations/img/deduplicate_row_to_keep.png
Comment (alexanderahn) [reply docs/SQL/gems/transformations/img/deduplicate_row_to_keep.png:1]: Ack.

File: docs/SQL/gems/transformations/deduplicate.md (hunk 1 above)
Comment (alexanderahn) [reply docs/SQL/gems/transformations/deduplicate.md]: Done.

**Review by prashantpiyush — APPROVED** [review]

File: docs/SQL/gems/transformations/deduplicate.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+| :--------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------- |
+| Source                 | Input source                                                                                                                                                                                                                                                                                                               | True     |
+| Row to keep            | - `Distinct Rows`: Keeps all distinct rows. This is equivalent to performing a `df.distinct()` operation <br/>- `Unique Only`: Keeps rows that don't have duplicates <br/>- `First`: Keeps first occurrence of the duplicate row <br/>- `Last`: Keeps last occurrence of the duplicate row <br/>Default is `Distinct Rows` | True     |
```
Comment (bandikishores) [inline docs/SQL/gems/transformations/deduplicate.md:20]: `df.distinct()` is for Spark. We can say equivalent to performing `select distinct ...`

File: docs/SQL/gems/transformations/deduplicate.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+| Deduplicate On Columns | Columns to consider while removing duplicate rows (not required for `Distinct Rows`)                                                                                                                                                                                                                                       | True     |
+
+## Row to keep options
+
+As mentioned in the previous parameters, there are four Row to keep options that you can use in your deduplicate Gem.
```
Comment (bandikishores) [inline docs/SQL/gems/transformations/deduplicate.md:25]: nit - Should we highlight the part in the statement?

As mentioned in the previous parameters, there are four "Row to keep" options that you can use in your deduplicate Gem.

File: docs/SQL/gems/transformations/deduplicate.md (hunk 2 above)
Comment (alexanderahn) [reply docs/SQL/gems/transformations/deduplicate.md:20]: Done.

File: docs/SQL/gems/transformations/deduplicate.md (hunk 3 above)
Comment (alexanderahn) [reply docs/SQL/gems/transformations/deduplicate.md:25]: Done.

**Pull request merged**